source/sql/alembic/versions/46168c7abc89_change_fare_zone_id_to_string.py

```diff
@@ -16,9 +16,9 @@
 from sqlalchemy.dialects import postgresql
 
 def upgrade():
-    op.execute("ALTER TABLE navitia.stop_point ALTER COLUMN fare_zone TYPE text;")
+    op.execute("ALTER TABLE navitia.stop_point ALTER COLUMN fare_zone TYPE text using fare_zone::text;")
```

Comment:
Downgrade  -1 followed  by Upgrade head works with/without "using fare_zone::text" added in the methode upgrade.